### PR TITLE
Add web.dev/fast

### PR DESCRIPTION
**Pull request by TheJaredWilcurt**
#### Summary

Add link

#### Motivation

Since MDN is just linking to other resources, this is a reputable and detailed link that should be included on web optimization.

#### Metadata

* [ ] Adds a new document
* [ ] Rewrites (or significantly expands) a document
* [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/guide/performance/index.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ When building modern Web apps and sites, it's important to make your content per
 - [Google PageSpeed Insights](https://pagespeed.web.dev/)
 - [Lighthouse](https://developers.google.com/web/tools/lighthouse/)
 - [WebPageTest](https://www.webpagetest.org/)
+- [Web.dev Guide](https://web.dev/fast/)
```
Comment (teoli2003) [inline files/en-us/web/guide/performance/index.md:15]:
```suggestion
- [Fast load times](https://web.dev/fast/) on _web.dev_
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Just a rewording to give the actual title

**Review by estelle — DISMISSED** [review]
The list is a list of tools. Web.dev/fast is a guide, not a tool. This would be a good addition as a "see also" as a reference to where you can find additional tools, but web.dev/fast should not be listed in the tools list.

**Comment by dipikabh** [comment]
Hello @TheJaredWilcurt, as pointed out by Estelle, could you move "[Fast load times](https://web.dev/fast/) on _web.dev_" out of the list of tools and move it to a new section "See Also"? We should then be able to close this PR. Thanks!

**Review by dipikabh — APPROVED** [review]

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**